### ISSUE TITLE
Add Disqus to DMC

### ISSUE THREAD

**Issue by luisherranz** [wp-pwa/saturn-theme#269]
shortname: "diariomotorcompeticion"

child of #243

**Comment by orballo** [comment]
Added to the database. We need to test it once they have the plugin working.

**Comment by luisherranz** [comment]
It's not working yet. I'm not sure why, maybe we have to check the parameters they are passing to their own disqus iframe and see if they match with ours

**Comment by luisherranz** [comment]
@orballo are you sure this is working? I can't seem to make it work.

**Comment by orballo** [comment]
Yup, it takes a while to load the iframe but it's working. Did you pull my changes?

**Comment by orballo** [comment]
![screenshot from 2018-05-03 19 56 04](https://user-images.githubusercontent.com/13615057/39594116-0c627a36-4f0c-11e8-9418-43f632fa370a.png)

**Comment by luisherranz** [comment]
Oh, no I didn't see your commit, sorry! My bad. Thanks @orballo

**Comment by orballo** [comment]
It would be great that waffle showed the commits related to the issue :/

**Comment by luisherranz** [comment]
Why do you say that?

**Comment by orballo** [comment]
So you can see that there're things done that aren't comments. I'm guessing you missed it because you checked the issue in waffle.

**Comment by luisherranz** [comment]
Oh, I see!
